_stbt/match.py

```diff
@@ -776,11 +789,17 @@ def _log_match_image_debug(imglog):
         </head>
         <body>
         <div class="container">
+        <a href="../{{ "%05d" % (frame_number - 1) }}/index.html"
+           class="nav pull-left">«prev</a>
+        <a href="../{{ "%05d" % (frame_number + 1) }}/index.html"
+           class="nav pull-right">next»</a>
         <h4>
+            {{"Matched" if matched else "Didn't match"}}
             <i>{{template_name}}</i>
-            {{"matched" if matched else "didn't match"}}
         </h4>
 
+        <img src="source_with_matches.png" />
```

Comment:
We could use divs here like we do with `ocr`, rather than drawing on the image itself.